README.md

```diff
@@ -64,26 +64,31 @@ upstream, maps, and global vars are to be defined by using either
 
 ### Upstream Vars
 `nginx_upstreams` is a list of dicts containing the following two entries.
-| Name   |      Required      | Default | Description                       |
-|:-------|:------------------:|:--------|:----------------------------------|
-| `name` | :heavy_check_mark: |         | Upstream name used in domain_vars |
-| `path` | :heavy_check_mark: |         | url or socket to php listener     |
+
+| Name      | Required           | Default | Description                                                                                                             |
```

Comment:
The default column is not really needed anywhere. You could merge Required and Default

Reply:
Yes i could or stick to what this role already has

Reply:
Of course, I've just seen you change it on existing roles and create PRs to reflect on better practices before and though you might want to do this here, too.